libbpf-rs/src/map.rs

```diff
@@ -731,3 +734,117 @@ impl<'a> Iterator for MapKeyIter<'a> {
         }
     }
 }
+/// Represent a wrapper `bpf_map_info` struct pointer
+/// Provide ability to retrive the details of a certain map with its fd only
+#[derive(Debug)]
+pub struct MapInfo {
+    info: bpf_map_info,
+}
+
+impl MapInfo {
+    /// Create a MapInfo struct from a fd.
+    pub fn new(fd: i32) -> Result<Self> {
+        let mut map_info = unsafe { std::mem::zeroed::<bpf_map_info>() };
```

Comment:
Please use `use` declarations at the top: `use std::mem` (here and everywhere). Please also add a [`SAFETY` comment](https://std-dev-guide.rust-lang.org/documentation/safety-comments.html#inside-safe-elements).

Reply:
Fixed now